api/core/v1alpha1/model_types.go

```diff
@@ -49,6 +49,12 @@ type ModelHub struct {
 	// +kubebuilder:default=main
 	// +optional
 	Revision *string `json:"revision,omitempty"`
```

Comment:
Let's update the comment of Filename as well, 
```
	// Filename refers to a specified model file rather than the whole repo.
	// This is helpful to download a specified GGUF model rather than downloading
	// the whole repo which includes all kinds of quantized models.
	// TODO: this is only supported with Huggingface, add support for ModelScope
	// in the near future.
        // Note: once filename is set, allowPatterns and ignorePatterns should be left unset.
```